packages/fuel-indexer-api-server/src/uses.rs

```diff
@@ -43,6 +44,11 @@ use std::{
 use tokio::sync::mpsc::Sender;
 use tracing::error;
 
+#[derive(Serialize)]
```

Comment:
This should go in api/models.rs

Reply:
I'll move it over in the final docs/cleanup PR today. 👌🏽